services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -415,4 +438,42 @@ export class SignifyApi {
     }
     return op;
   }
+
+  async saidifySchema(schema: any, label?: string): Promise<any> {
```

Comment:
The label for schema is always $id or whatever. So it shouldn't be an argument here or in `saveSchema`.